web/satellite/src/components/infoBars/PaidTierBar.vue

```diff
@@ -6,20 +6,30 @@
         <p>
             Upload up to 75TB by upgrading to a Storj Pro Account.
         </p>
-        <p class="pt-bar__functional" @click.stop="openAddPMModal">
+        <p class="pt-bar__functional" @click.stop="openAddPMModal" @click="openBanner">
             Upgrade now.
         </p>
     </div>
 </template>
 
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator';
+import { AnalyticsHttpApi } from '@/api/analytics';
+import { AnalyticsEvent } from '@/utils/constants/analyticsEventNames';
 
 // @vue/component
 @Component
 export default class PaidTierBar extends Vue {
     @Prop({default: () => false})
     public readonly openAddPMModal: () => void;
+    private readonly analytics: AnalyticsHttpApi = new AnalyticsHttpApi();
+
+    // Send analytics event to segment when Upgrade Account banner is clicked.
+    public async openBanner(): Promise<void> {
+ 
+        await this.analytics.eventTriggered(AnalyticsEvent.UPGRADE_BANNER_CLICKED);
```

Comment:
yeah, so just add `this.openAddPMModal()` here and remove redundant @click handler from template please